### PR TITLE
Provider labels for AWS, Azure and DO nodes

### PR DESCRIPTION
**Pull request by kke**

### CODE REVIEW

File: cli/lib/kontena/machine/digital_ocean/node_provisioner.rb [hunk 1]
```diff
@@ -47,7 +47,14 @@ def run!(opts)
           ShellSpinner "Waiting for node #{droplet.name.colorize(:cyan)} join to grid #{opts[:grid].colorize(:cyan)} " do
             sleep 2 until node = droplet_exists_in_grid?(opts[:grid], droplet)
           end
-          set_labels(node, ["region=#{opts[:region]}", "az=#{opts[:region]}"])
+          set_labels(
+            node,
+            [
+              "region=#{opts[:region]}",
+              "az=#{opts[:region]}",
+              "provider=digital_ocean"
```
Comment (nevalla) [inline cli/lib/kontena/machine/digital_ocean/node_provisioner.rb]: @jakolehm @jnummelin any opinions what provider name we should use for DigitalOcean?

Comment (jakolehm) [inline cli/lib/kontena/machine/digital_ocean/node_provisioner.rb]: maybe `digitalocean` ?

Comment (jnummelin) [inline cli/lib/kontena/machine/digital_ocean/node_provisioner.rb]: cli commands are named with `digitalocean`, maybe follow that naming throughout?

**Comment by jakolehm** [comment]
`digitalocean` provider name got most votes.. can you make that change @kke ? Otherwise looks good!

**Pull request merged**